homeassistant/components/plugwise/climate.py

```diff
@@ -117,6 +110,21 @@ def hvac_mode(self) -> HVACMode:
             return HVACMode.HEAT
         return HVACMode(mode)
 
+    @property
+    def hvac_modes(self) -> list[HVACMode]:
+        """Return the list of available HVACModes."""
+        hvac_modes: list[HVACMode] = [HVACMode.HEAT]
+        if self.coordinator.data.gateway["cooling_present"]:
+            hvac_modes = [HVACMode.HEAT_COOL]
+
+        if self.device["available_schedules"] != ["None"]:
+            if HVACMode.AUTO not in hvac_modes:
+                hvac_modes.append(HVACMode.AUTO)
+        elif HVACMode.AUTO in hvac_modes:
+            hvac_modes.remove(HVACMode.AUTO)
```

Comment:
```suggestion
```

Reply:
Sorry, this I don't understand. We should not remove HVACMode.AUTO when there's no schedule defined?

I mean the full logic for these two line reads like:
```
        if self.device["available_schedules"] == ["None"]:
            if HVACMode.AUTO in hvac_modes:
                hvac_modes.remove(HVACMode.AUTO)
```
When the available_schedules-list only contains "None", meaning there is no schedule defined, then the HVACMode.AUTO is no longer available and should be removed.

Reply:
> One or more schedules were define before and the user decides to remove them all.

There is no such case. `havc_modes` is fully created inside this method, the `elif` will never render true.